grape/general_graph.py

```diff
@@ -88,12 +68,36 @@ def load(self, filename):
             elif Type == "USER":
                 self.USER.append(idx)
 
-        self.valv = {	"isolation_A" : { "0": "OPEN", "1": "CLOSED"},
-			"isolation_B" : { "0": "CLOSED", "1": "OPEN"},
-			"unknown" : { "0": "OFF", "1": "ON"} }
-
         return graph_df, graph_edges_df
 
+    @property
+    def area(self):
+        return nx.get_node_attributes(self, 'Area')
+
+    @property
+    def Fault_Resistant(self):
+        return nx.get_node_attributes(self, 'PerturbationResistant')
+
+    @property
+    def Description(self):
```

Comment:
Python naming convention https://www.python.org/dev/peps/pep-0008/#naming-conventions: no capital for methods/attributes